proposals/0256-Refactor-Fuel-Information-Related-Vehicle-Data.md

```diff
@@ -128,13 +148,13 @@ We need to:
             The estimate range in KM the vehicle can travel based on fuel level and consumption.
         </description>
     </param>
-+   <param name="level" type="Float" minvalue="-6" maxvalue="170" mandatory="false">
++   <param name="level" type="Float" minvalue="-6" maxvalue="1000" mandatory="false">
```

Comment:
```suggestion
+   <param name="level" type="Float" minvalue="-6" maxvalue="1000000" mandatory="false">
```

Reply:
Please see [comment](https://github.com/smartdevicelink/sdl_evolution/issues/854#issuecomment-549878857), `level` is 10000. I believe when you were noting down in meeting then we agreed to this comment. 10000 is anyways an overkill for percentage itself. But since you have noted down 1000000 as official decision from meeting, i can update it to that still. Please let me know.

Reply:
We should update to the official decision from the meeting, thanks!

Reply:
Done